src/controllers/AuthController.ts

```diff
@@ -0,0 +1,14 @@
+import { Request, Response } from 'express'
+import { verifyToken } from '../helpers/AuthHelper'
+
+const verifyTokenController = async (req: Request, res: Response) => {
+  const { token } = req.body
+  try {
+    await verifyToken(token)
+    res.status(200).send({ msg: 'authorized' })
```

Comment:
plz don't use such a messy shortcuts, just name it message. One more thing it should be `{ 'message' : 'success', 'user': {...} }` here and `{ 'error': 'unauthorized'}`.